src/lib.rs

```diff
@@ -1,6 +1,11 @@
-use core::panic;
-use std::cmp;
 #[doc = include_str!("../README.md")]
```

Comment:
My bad, I accidentally typo'd myself last time. This needs to be written as the following:
```rs
#![doc = include_str!("../README.md")]
```